AmplicaLabs/wallet-proxy
# Handle selection

# Select account and choose a handle, part 2/2
* puts wallet info into a Svelte store
* puts the handle choice into a Svelte store
* Fixes some small issues with setting state between components
* Removes the 'claim handle' button since we're saving all the signing for last

https://github.com/AmplicaLabs/wallet-proxy/assets/502640/844d5afb-f6ce-4780-9757-d4947711f7df

Closes #16
Closes #33 

To Verify:
* `npm run start`
* Go to http://localhost:5173 and select the endpoint + type that you want to test.
* Verify that you can complete all steps through Register, to get signatures for creating an MSA + delegation, and a Handle.
